### ISSUE TITLE
Example request / reply doesn't work (tokio 1.0)

### ISSUE THREAD

**Issue by ev94** [cetra3/tmq#21]
Hi,

Thx for the great work. We really appreciate it. We are now moving our project to tokio 1.0.
I've seen on master that you are doing the same. I've just clone it. It seems that the example request / reply doesn't work. The request is sent. But the server never get it. The method ```recv_sock.recv().await?``` doesn't return messages.

Is the migration of tmq to tokio 1.0 still in progress?

Request side:

```
tmq git:(master) ✗ RUST_LOG=debug cargo run --example request
    Finished dev [unoptimized + debuginfo] target(s) in 0.04s
     Running `target/debug/examples/request`
 INFO  request > Request: "Req#0"
```

Reply side:

```
tmq git:(master) ✗ RUST_LOG=debug cargo run --example reply
   Compiling tmq v0.2.1 (/home/emmanuel/dev/tmq)
    Finished dev [unoptimized + debuginfo] target(s) in 1.13s
     Running `target/debug/examples/reply`
 INFO  reply > connecting
 INFO  reply > connected
 INFO  reply > waiting for message
```

**Comment by skrap** [comment]
See #20, perhaps related.

**Comment by ev94** [comment]
Just tried YushiOMOTE fix #20. 
It fixes the issue.

**Comment by cetra3** [comment]
Resolved by #20
